maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/NewMavenJSFModuleProjectProvider.java

```diff
@@ -277,13 +277,14 @@ protected IPath createArchetypeProject(NewLiferayJSFModuleProjectOp op, IProgres
 			Exception cause = result.getCause();
 
 			if (cause != null) {
-				throw new CoreException(LiferayCore.createErrorStatus("Unable to create project from archetype."));
+				throw new CoreException(
+					LiferayCore.createErrorStatus("Unable to create project from archetype.", cause.getMessage()));
```

Comment:
Send the entire exception instead of just the message.  

`createErrorStatus("", cause.getMessage(), cause)`